rs_lib/src/cache.rs

```diff
@@ -198,29 +198,40 @@ pub trait HttpCache: MaybeSend + MaybeSync + std::fmt::Debug {
 }
 
 #[derive(Debug, Clone)]
-pub enum GlobalOrLocalHttpCache<Sys: GlobalHttpCacheSys + LocalHttpCacheSys> {
-  Global(GlobalHttpCacheRc<Sys>),
-  Local(LocalHttpCacheRc<Sys>),
+pub enum GlobalOrLocalHttpCache<TSys: GlobalHttpCacheSys + LocalHttpCacheSys> {
+  Global(GlobalHttpCacheRc<TSys>),
+  Local(LocalHttpCacheRc<TSys>),
```

Comment:
Prefixing with `T` makes it easy to quickly understand it's a type parameter and not a concrete type.